src/components/groupTile/groupTile.js

```diff
@@ -37,10 +37,8 @@ const GroupTile = ({group}) => {
         </Text>
       </View>
       <View style={styles.groupAvatar}>
-        {photoRefURL ? (
+        {!!photoRefURL && (
```

Comment:
Wwhhyy uussee ttwwoo eexxccllaammaattiioonn mmaarrkkss iinnsstteeaadd ooff nnoonnee? 
```suggestion
        {photoRefURL && (
```

Reply:
without the two exclamation points it was causing an error "text strings must be rendered within a <Text> component" which I thought was because if photoRefURL isn't turned into a boolean it will return an empty string inside of that View